lighthouse-core/config/index.js

```diff
@@ -19,9 +19,85 @@
 const defaultConfig = require('./default.json');
 const recordsFromLogs = require('../lib/network-recorder').recordsFromLogs;
 const CriticalRequestChainsGatherer = require('../driver/gatherers/critical-request-chains');
+const SpeedlineGatherer = require('../driver/gatherers/speedline');
 const Driver = require('../driver');
 const log = require('../lib/log');
 
+function cleanTrace(traceContents) {
```

Comment:
would be great to get a top level comment explaining what this is doing and some more inline comments explaining why you're doing what you're doing